src/oge/emissions.py

```diff
@@ -223,8 +223,18 @@ def identify_geothermal_generator_geotype(year):
     return geothermal_geotype
 
 
-def adjust_fuel_and_emissions_for_CHP(df):
-    """Allocates total emissions for electricity generation."""
+def adjust_fuel_and_emissions_for_CHP(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Name of function should be in lowercase: CHP -> chp

Reply:
Fixed